.github/workflows/units-test-scripts-structures-pruning.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [ '3.8', '3.12' ]
+        python-version: [ '3.8', '3.11', '3.12' ]
```

Comment:
is 3.12 purposeful?

Reply:
When the changes were coming into edx-platform we still thought we were going directly to 3.12 so that's why it was there in the first place. I left 3.12 in since otherwise I'll just have to add it in a future PR and this one is small enough that I'm not worried.

Reply:
👍